test/cmd/dnshammer.go

```diff
@@ -115,15 +118,45 @@ func compare(a, b []string) bool {
 	return true
 }
 
-func lookupARecord(addr, resolverProtocol, domain string) ([]net.IP, error) {
-	resolver := net.Resolver{
-		Dial: func(ctx context.Context, network, address string) (net.Conn, error) {
-			dialer := net.Dialer{}
-			return dialer.DialContext(ctx, resolverProtocol, addr)
-		},
+func lookupARecord(domain string) (out []string, err error) {
+	ips, err := net.LookupIP(domain)
+	for _, ip := range ips {
+		out = append(out, ip.String())
+	}
+	return out, err
+}
+
+func loadRecords(p string) map[string][]string {
+	path := filepath.ToSlash(p)
+	f, err := os.Open(path)
+	if err != nil {
+		logrus.Panicf("opening file: %v err: %v", path, err)
+	}
+	defer func() {
+		if err := f.Close(); err != nil {
+			logrus.Errorf("loadRecords closing file: %v", err)
+		}
+	}()
+
+	csvReader := csv.NewReader(f)
+	csvReader.FieldsPerRecord = -1 // disable expected fields per record
+	fileData, err := csvReader.ReadAll()
+	if err != nil {
+		logrus.Panicf("reading csv file: %v err: %v", path, err)
+	}
+
+	var domain string
+	records := make(map[string][]string)
+	for _, line := range fileData {
+		var data = []string{}
+		for j, field := range line {
+			if j == 0 {
+				domain = field
+			} else {
+				data = append(data, field)
+			}
+			records[domain] = data
+		}
```

Comment:
I just realized… isn't this the same as:
```suggestion
		if len(line) > 0 {
			records[line[0]] = line[1:]
		}
```